data/apps/awair.json

```diff
@@ -0,0 +1,81 @@
+{
+  "info": {
+    "identifier": "awair",
+    "name": "Awair - Breathe Easy",
+    "appstoreUrl": "https://itunes.apple.com/app/id1034660006"
+  },
+  "actions": [
+    {
+      "name": "list",
```

Comment:
I didn't really create a style guide for the documentation or anything, but it'd be really cool if the names could be in title case. And maybe the description could be a bit more descriptive like "Opens the device list view".